www/include/reporting/dashboard/viewServicesLog.php

```diff
@@ -137,7 +135,7 @@
 /*
  * Stats Display for selected service
  */
-if (isset($host_id) && $host_id != "NULL" && isset($service_id) && $service_id != "NULL") {
+if ($host_id !== false && $service_id !== false) {
```

Comment:
as this condition is the same than the previous one, you should concatenate them to avoid superfuous operations

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/23185b9036383d869664a0590531e69abe7f06c2